keras2onnx/_builtin.py

```diff
@@ -553,6 +553,10 @@ def convert_tf_depth_to_space(scope, operator, container):
     node = operator.raw_operator
     block_size = node.get_attr('block_size')
     oopb = OnnxOperatorBuilder(container, scope)
+    if container.target_opset < 13:
```

Comment:
which op will use this op_version? should we check the target_opset lower bound as well?

Reply:
synced offline.